tests/test_dist_metrics.py

```diff
@@ -64,3 +65,25 @@ def test_tvd_one_empty_dist():
     target_bsd[bs2] = 0.7
 
     assert tvd_dist(target_bsd, BSDistribution()) == 0.5
+
+
+def test_kl_div_identical_dist():
+    target_bsd = BSDistribution()
+    target_bsd[bs1] = 0.5
+    target_bsd[bs2] = 0.5
+
+    bsd_to_comp = copy(target_bsd)
+
+    assert kl_divergence(target_bsd, bsd_to_comp) == 0
+
+
+def test_kl_div_unequal_dist():
+    ideal_bsd = BSDistribution()
+    ideal_bsd[bs1] = 0.25
+    ideal_bsd[bs2] = 0.5
+
+    model_bsd = BSDistribution()
+    model_bsd[bs1] = 1
+    model_bsd[bs2] = 0.7
+
+    assert kl_divergence(ideal_bsd, model_bsd) != 0
```

Comment:
Isn't != a bit too broad? Isn't there an easy expected value between two distributions that we could use to check the result value?